### PR TITLE
Dev imagemeshing

### PR DESCRIPTION
**Pull request by JacobTh98**
With this function it is possible to convert a 200x200 pixel image to the unit circle pyEITmesh. Any geometry can be generated as an image and processed into the relating mesh. The pixels with 0 are the empty space and with 1 are object. A preview is shown inside the Readme.md:
https://github.com/JacobTh98/pyEIT_image_wrapper

### CODE REVIEW

**Comment by liubenyuan** [comment]
Please correct the black linting errors.
The requirements under `pyeit/mesh` folder is not required and the self-requirement on `pyeit` is not required. If your package requires a specific package, a minimal version should be provided as is.

**Comment by JacobTh98** [comment]
Excuse the circumstances. It's my first pull request. I will check the whole thing with the CI and then submit it again.

File: pyeit/mesh/mesh_img.py [hunk 1]
```diff
@@ -1,22 +1,28 @@
-# The wrapper function to transform a 200x200 picture to the pyEIT mesh.
-# For examples see: https://github.com/JacobTh98/pyEIT_image_wrapper
-# With pyeit version <1.2.0 look on the linked repository too.
-
 import numpy as np
-import pyeit.mesh as mesh
-from pyeit.mesh import create
+#Was üassoe
```
Comment (liubenyuan) [inline pyeit/mesh/mesh_img.py]: This line can be deleted.

**Comment by liubenyuan** [comment]
Hi, `pyeit` use black and flake as the default linter. You can install black and flake8 using
```bash
$ conda install black flake8
$ black .
$ flake8
```
and to check your linting CI.

**Comment by liubenyuan** [comment]
May be a wrong black version? This github workflow use the black version 22.3, it might have different formatting rules to your version.

**Pull request closed without merging**